test/suites/integration/bsp/multiple-delete.test.ts

```diff
@@ -81,6 +81,9 @@ describeBspNet("Single BSP Volunteering", ({ before, createBspApi, it, createUse
     await sleep(500);
     await userApi.wait.bspStored(1);
 
+    // Wait for BSP to update its local Forest root before starting to generate the inclusion proofs
+    await sleep(2000);
```

Comment:
We could maybe have a `wait` that waits for certain file keys to be in the forest storage of the BSP (which should be enough to guarantee that the forest root has been updated)